service/app/validators/CertVerifyRequestValidator.java

```diff
@@ -0,0 +1,47 @@
+package validators;
+
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.BaseException;
+import org.sunbird.JsonKeys;
+import org.sunbird.message.IResponseMessage;
+import org.sunbird.message.ResponseCode;
+import org.sunbird.request.Request;
+
+import java.text.MessageFormat;
+import java.util.Map;
+
+/**
+ * this class will be used to validate the cert verify API request.
+ * @author anmolgupta
+ */
+public class CertVerifyRequestValidator implements IRequestValidator {
```

Comment:
I think you can write a simple test class for this. Such incremental and small changes matters a lot later.

Reply:
@indrajra done